clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1983,6 +1983,17 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
   }];
 }
 
+def ReqdWorkGroupSizeAttrDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "cl::reqd_work_group_size, intel::reqd_work_group_size";
```

Comment:
What about GNU spelling?

Reply:
I don't sure about it. I will ask about it.

Reply:
The answer of John Pennycook is following:
"I hadn't thought about this before, but I'm going to say "No" – at least for now.  If we're not using the C++11 attribute specifier syntax we can't have a namespace, and I don't want to set a precedent for converting between intel:: and intel_ that might accidentally introduce name conflicts."

Reply:
So that a GNU spelling doesn't need here.

Reply:
@Pennycook, please note that this is documentation for the compiler itself, not for the particular language. It is used to generated the following documentation: https://clang.llvm.org/docs/AttributeReference.html

So, this attribute is accepted by the compiler in GNU spelling and therefore, must be documented here as well if you adding such documentation. In SYCL-specific documents we can omit GNU spelling, but not here.

BTW, looking at the web-page I referenced above, it would be okay for me to either mention all declared spellings or just leave `reqd_work_group_size` generic string

Reply:
@fadeeval, this comment is the only one I have. I would like header to be aligned with the actual documentation, which describes all three spellings

Reply:
Ok.